jsonnetsecure/jsonnet_processvm.go

```diff
@@ -26,10 +28,19 @@ func NewProcessVM(opts *vmOptions) VM {
 }
 
 func (p *ProcessVM) EvaluateAnonymousSnippet(filename string, snippet string) (string, error) {
+	ctx, span := trace.SpanFromContext(p.ctx).TracerProvider().Tracer("").Start(p.ctx, "jsonnetsecure.ProcessVM.EvaluateAnonymousSnippet")
+	defer span.End()
+
 	// We retry the process creation, because it sometimes times out.
 	const processVMTimeout = 1 * time.Second
-	return backoff.RetryWithData(func() (string, error) {
-		ctx, cancel := context.WithTimeout(p.ctx, processVMTimeout)
+	return backoff.RetryWithData(func() (_ string, err error) {
+		defer func() {
+			if err != nil {
+				span.AddEvent("retry", trace.WithAttributes(attribute.String("error", err.Error())))
```

Comment:
Not sure if event is the right thing here, but seems to make sense. I can also add `err_0`, `err_1`, ... as attributes for each iteration.